fx/native_interpreter/README.md

```diff
@@ -0,0 +1,29 @@
+# Lowering PyTorch Code to a Native Runtime With FX and TorchScript Custom Classes
+
+In this example, we are going to build an FX transformation takes the code from a PyTorch `Module` and converts (aka “lowers”) it to a format that a native C++ runtime can consume and execute. We are going to build up a trivial interpreter for this example, but you can imagine extending the same process to work with more sophisticated backends, ones which may do code optimization or offloading to an accelerator.
```

Comment:
I think there is more going on here than just a lowering, this example shows how to

1. Lower the FX graph to some other representation
2. Register that other representation as a custom operator that can be used from TorchScript or Python.
3. Wrap the lowered code into a format that can still be used within TorchScript.

Lots of things are simpler if you don't need to do 1--3, so a possible improvement would be to justify each component with whether it is for 1, 2, or 3 above.  I feel like without this clarification, this seems a lot more complicated than if someone, e.g., wanted to write a quick exporter to another format.